util/color.zig

```diff
@@ -20,6 +21,7 @@ pub const Color = enum {
             .magenta => "\x1b[35m",
             .cyan => "\x1b[36m",
             .reset => "\x1b[0m",
+            .bold => "\x1b[1m",
```

Comment:
"color" is kind of weird here ^^ These are all font styles, no?

Reply:
True, thats weird. Do you have any recommendations for it? Rename to `Font` or `FontStyle`?

Reply:
'FontStyle' seems concise to me :)

Reply:
`TextStyle`?

Reply:
even better!